datasets/cellpainting-gallery.yml

```diff
@@ -38,13 +47,61 @@ DataAtWork:
       URL: https://github.com/carpenterlab/2016_bray_natprot/wiki
       AuthorName: Carpenter-Singh Lab
       AuthorURL: https://carpenter-singh-lab.broadinstitute.org/
+    - Title: Image-based Profiling Handbook - for processing image-based profiling datasets using CellProfiler and pycytominer
+      URL: https://cytomining.github.io/profiling-handbook/
+      AuthorName: Carpenter-Singh and Cimini Labs
+      AuthorURL: https://github.com/cytomining/profiling-handbook/graphs/contributors
+    - Title: Scientific Community Image Forum - for software-oriented aspects of scientific imaging including analysis, processing, and acquisition
+      URL: https://forum.image.sc
+      AuthorName: Center for Open Bioimage Analysis (COBA)
+      AuthorURL: https://openbioimageanalysis.org
+    - Title: Center for Open Bioimage Analysis (COBA) YouTube Channel - video tutorials of CellProfiler and other softwares
+      URL: https://www.youtube.com/c/COBACenterforOpenBioimageAnalysis/videos
+      AuthorName: Center for Open Bioimage Analysis (COBA)
+      AuthorURL: https://openbioimageanalysis.org
   Tools & Applications:
     - Title: Pycytominer - Data processing functions for profiling perturbations
       URL: https://github.com/cytomining/pycytominer
-      AuthorName:
+      AuthorName: Carpenter-Singh Lab
       AuthorURL: https://github.com/cytomining/pycytominer/graphs/contributors
+    - Title: Distributed CellProfiler - Run encapsulated docker containers with CellProfiler in the Amazon Web Services infrastructure
+      URL: https://github.com/CellProfiler/Distributed-CellProfiler
+      AuthorName: Cimini Lab
+      AuthorURL: https://github.com/CellProfiler/Distributed-CellProfiler/graphs/contributors
+    - Title: Deep Profiler - Morphological profiling using deep learning
+      URL: https://github.com/cytomining/DeepProfiler
+      AuthorName: Carpenter-Singh Lab
+      AuthorURL: https://github.com/cytomining/DeepProfiler/graphs/contributors
+    - Title: Image-based Profiling Recipe
+      URL: https://github.com/cytomining/profiling-recipe
+      AuthorName: Carpenter-Singh Lab
+      AuthorURL: https://github.com/cytomining/profiling-recipe/graphs/contributors
   Publications:
     - Title: Cell Painting, a high-content image-based assay for morphological profiling using multiplexed fluorescent dyes
       URL: https://pubmed.ncbi.nlm.nih.gov/27560178/
-      AuthorName: Bray M-A, Singh S, Han H, Davis CT, Borgeson B, Hartland C, Kost-Alimova M, Gustafsdottir SM, Gibson CC, & Carpenter AE.
+      AuthorName: Bray M-A, Singh S, Han H, Davis CT, Borgeson B, Hartland C, Kost-Alimova M, Gustafsdottir SM, Gibson CC, & Carpenter AE
+      AuthorURL: https://carpenter-singh-lab.broadinstitute.org/
```

Comment:
For papers, I'd favor dropping `AuthorURL` in favor of `AuthorName`, because the latter is always more accurate

Reply:
If we must fill `AuthorURL`, I'd go with "Multiple"